engine/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -57,45 +37,31 @@ object MetadataServiceActor {
   case class MetadataLookupResponse(query: MetadataQuery, eventList: Seq[MetadataEvent]) extends MetadataServiceResponse
   case class MetadataServiceKeyLookupFailed(query: MetadataQuery, reason: Throwable) extends MetadataServiceResponse
 
-  def props(serviceConfig: Config, globalConfig: Config) = {
-    Props(MetadataServiceActor(serviceConfig, globalConfig))
+  def props(dataAccess: DataAccess, serviceConfig: Config, globalConfig: Config) = {
+    Props(MetadataServiceActor(dataAccess, serviceConfig, globalConfig))
   }
 }
 
-case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config) extends Actor {
+case class MetadataServiceActor(dataAccess: DataAccess, serviceConfig: Config, globalConfig: Config) extends Actor {
```

Comment:
This used to be a field in the MetadataServiceActor and I was asked to make it a constructor parameter.  The engine currently does not use a DataAccess itself.  Not sure what I should do at this point.  😦